docs/topics/migrations.txt

```diff
@@ -476,6 +476,30 @@ You can pass a second callable to
 want executed when migrating backwards. If this callable is omitted, migrating
 backwards will raise an exception.
 
+.. note::
+
+    When migrating data across models spanning more than one app, ``dependencies``
```

Comment:
maybe "When writing a `RunPython` function that uses models from more than one app, " would be clearer wording.